framework/db/mssql/QueryBuilder.php

```diff
@@ -25,6 +25,7 @@ class QueryBuilder extends \yii\db\QueryBuilder
     public $typeMap = [
         Schema::TYPE_PK => 'int IDENTITY PRIMARY KEY',
         Schema::TYPE_BIGPK => 'bigint IDENTITY PRIMARY KEY',
+        Schema::TYPE_CHAR => 'char(1)',
```

Comment:
Noted. Think I may redo this branch and leave the tests in the horizontal format.